### PR TITLE
Exposing timeout parameter for packs.install action to help with long running installs.

### PR DESCRIPTION
**Pull request by hnanchahal**
Some packs can get a timeout after trying to set up virtual env for 600 sec. This can happen if someone is using persistent storage outside and there is latency in read/write. The default action timeout for packs.install action is controlled by the default timeout for the python_script action runner which is 600 sec. This change will allow to override the timeout.

### CODE REVIEW

**Review by armab — CHANGES_REQUESTED** [review]
That's really helpful @hnanchahal, thanks for the contribution! :+1: 

Everything looks good, we just need a changelog record for this enhancement. See CHANGELOG.rst for the examples and format to follow.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -28,6 +28,11 @@ Added
 
   Contributed by @amanda11
 
+* Added timeout parameter for packs.install action to help with long running installs that exceed the 
+  default timeout of 600 sec which is defined by the python_script action runner (improvement)
```
Comment (armab) [inline CHANGELOG.rst:32]:
```suggestion
  default timeout of 600 sec which is defined by the python_script action runner (improvement) #5047
```

Comment (hnanchahal) [reply CHANGELOG.rst]: Added

**Review by armab — APPROVED** [review]
Looks good :+1: 

Thanks for the contribution!

**Comment by armab** [comment]
@hnanchahal The only missing thing to merge the PR is CLA signing process, see https://github.com/StackStorm/st2/pull/5047#issuecomment-698493505 above for the instructions.

File: contrib/packs/actions/workflows/install.yaml [hunk 2]
```diff
@@ -98,6 +99,7 @@ tasks:
       packs: <% ctx().packs_list %>
       env: <% ctx().env %>
       python3: <% ctx().python3 %>
```
Comment (nmaludy) [inline contrib/packs/actions/workflows/install.yaml:101]: i'm wondering if this timeout should also be applied to other sub-actions such as the pack content registration (thinking EC2 pack where there is 1000+ actions)

**Review by nmaludy — COMMENTED** [review]
Question left inline about timeout parameter usage.

**Comment by nmaludy** [comment]
@hnanchahal this PR is currently held up because you haven't signed our CLA (Contributor License Agreement), can you please do so and we'll be happy to merge this PR!

**Comment by nmaludy** [comment]
Removing Milestone until CLA is signed

**Comment by hnanchahal** [comment]
> Removing Milestone until CLA is signed

The old account hnanchah no longer has a backing email. That email is no longer valid so i am unable to add that as a secondary email and validate. Any other alternatives? Do you want me to submit a new PR?

**Comment by blag** [comment]
If we just open a new PR, if you just cherry-pick the commits into a new branch and push, the commits will still be authored (in Git) by the now-invalid "hnanchah" account. I believe that we will need to generate brand new commits from scratch by manually copying and pasting the code between branches (eg: no simple `git cherry-pick` commits) and open a brand new PR with that branch.

@hnanchahal Can you not even login to the old "hnanchah" GitHub account? Is it tied into TFA or an old SSO or something?

**Comment by hnanchahal** [comment]
CLA Issues. Will raise a duplicate PR.

**Pull request closed without merging**